### ISSUE TITLE
Consider reusing documentation developed by friendly OpenHisiIpCam project 

### ISSUE THREAD

**Issue by pfalcon** [OpenIPC/openipc.github.io#9]
https://github.com/OpenHisiIpCam/ is a "sister" (brother?) project developed by people who also participate in OpenIPC telegram channels and otherwise cooperate with OpenIPC maintainers.

The current situation (from outsider point of view):

* OpenIPC seems to have collected some critical mass in regard to firmware hacking/development, and has firmware releases for a number of devices.
* At the same time, OpenHisiIpCam's repos seem to have been updated ~1 year ago (it seems the project went into "stealth startup" mode, which is otherwise quite understood).
* On the other hand, OpenHisiIpCam has something which can be called "manual", hosted at https://www.openhisiipcam.org/
* While for OpenIPC, there're only disparate, spread-around "notes" covering various topics, but not representing a coherent documentation (in particular, not suitable for beginners, which need some smooth introduction into the area).
* Yet, it's clear the the matter of documentation becomes the pressing one for the OpenIPC project, as it starts to attract more and more users.

Based, on that, there was an idea to ask OpenHisiIpCam's maintainers (@nikitos1550) if there's any chance at all that they could contribute the existing docs of the OpenHisiIpCam for reuse and further development in OpenIPC project. Likewise, there's a question to OpenIPC maintainers, whether they would be interested to accept such docs into the project, perform initial update for the OpenIPC and integration into the project infrastructure, and maintain them going forward.

This question really should be clarified before attempting to write any docs for OpenIPC "from scratch".

**Comment by pfalcon** [comment]
Based on the discussion at the "OpenIPC modding" channel, the initial response from @nikitos1550 was positive. In particular, he clarified that docs sources are available in the following branch: https://github.com/OpenHisiIpCam/openhisiipcam.github.io/tree/src (`src` branch of https://github.com/OpenHisiIpCam/openhisiipcam.github.io). So, next steps would be to confirm whether OpenIPC maintainers (@ZigFisher, @widgetii) are interested and clarify license.


For reference, snapshots of discussion on the channel (in Russian):

> @pfalcon Автору, т.е. Никите @nikita1550, задавался ли вопрос, нет ли желания за'contribute'ить эти доки под umbrella OpenIPC? Если нет, ничего страшного, надо рожать свое.

> @nikitos1550 Morning, если вопрос про скопипастить, то проблем нет. Что-то дописать, у меня, пока что-то совсем плохо со временем (

> Morning. И спасибо за ответ. Вопрос скорее не про "скопипастить", а скорее про "возможно ли, чтобы проект OpenHisiIpCam открыл свои доки (которые сейчас на https://www.openhisiipcam.org/) для проекта OpenIPC, с целью базирования док проекта OpenIPC на этих доках". Вопрос щекотливый, но прежде чем кто-то захочет писать доки с нуля для OpenIPC, он захочет такой вопрос задать. Ответ "нет" разумеется вполне тоже будет понятен.

> @nikitos1550
> Речь про какие-то скрытые доки?
> Там даже исходники сайта же есть

> @pfalcon я не видел исходников на https://www.openhisiipcam.org/ , сайт пишет "powered by MkDocs", в репе лежат HTML'ы. Также вопрос лицензии.

> @nikitos1550 В репе два бранча

**Comment by pfalcon** [comment]
cc @OnlyDeniko

**Comment by OnlyDeniko** [comment]
> cc @OnlyDeniko

Как я с этим связан?)

**Comment by pfalcon** [comment]
> Как я с этим связан?)

If you're not Denis Kulandin from the OpenIPC telegram channel, apogies, and click "Unsubscribe" button on the right.

**Comment by OnlyDeniko** [comment]
> > Как я с этим связан?)
> 
> If you're not Denis Kulandin from the OpenIPC telegram channel, apogies, and click "Unsubscribe" button on the right.

Yes, I am Denis Kulandin, but I don't have any subscribe in openIPC telegram channel

**Comment by pfalcon** [comment]
> Yes, I am Denis Kulandin, but I don't have any subscribe in openIPC telegram channel

Apologies again.

Instead, cc: @DeeKey
